Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -1264,6 +1268,44 @@ <h3>
     schließen kann. Weiteren Angaben über Ihre Nutzung der Corona-Warn-App erhält der Anbieter Ihres
     Betriebssystems jedoch nicht.
 </p>
+<h3>
+    o. Aktualisierungsfunktion für COVID-Zertifikate
+</h3>
+<p>
+    Es kann vorkommen, dass die App Ihnen die Aktualisierung eines gültigen digitalen
+    COVID-Zertifikats anbietet. Beschließen die EU-Mitgliedstaaten beispielsweise eine Änderung des
+    Inhalts oder der Datenstruktur der Daten im COVID-Zertifikat, kann es für die reibungslose
+    weitere Zertifikatsnutzung erforderlich sein, die jeweilige inhaltliche oder technische Änderung
+    auf ein vorhandenes digitales COVID-Zertifikat anzuwenden.
+</p>
+<p>
+    Dazu lädt die App im Hintergrund die aktuellen inhaltlichen und technischen Spezifikationen für
+    die digitalen COVID-Zertifikate vom Serversystem der App herunter. Die App überprüft dann anhand
+    der in den von Ihnen gespeicherten digitalen COVID-Zertifikaten enthaltenen Daten, ob eine
+    Aktualisierung eines gespeicherten digitalen COVID-Zertifikats über die App möglich ist. Sie
+    werden dann auf eine mögliche Aktualisierung hingewiesen. Für die Aktualisierung wird mit Ihrem
+    Einverständnis das zu aktualisierende digitale COVID-Zertifikat von der App verschlüsselt an das
+    Serversystem der App übermittelt. Wenn es sich bei dem zu aktualisierenden digitalen
+    COVID-Zertifikat um ein COVID-Impfzertifikat zum Nachweis einer Auffrischungsimpfung
+    (Booster-Impfung) handelt, werden dabei auch die digitalen COVID-Zertifikate zum Nachweis der
+    der Auffrischungsimpfung vorausgegangenen Grundimmunisierung bzw. Genesung an das Serversystem
+    der App übermittelt. Dort wird dann die Gültigkeit der übermittelten digitalen COVID-Zertifikate
+    geprüft (siehe Abschnitt 6.h.). Anhand der Daten aus den übermittelten digitalen
+    COVID-Zertifikaten wird eine neue Version des COVID-Impfzertifikats für die Auffrischungsimpfung
+    generiert. Bei der Aktualisierung werden die Daten in den übermittelten COVID-Zertifikaten
+    (siehe Abschnitte 5.h und 5.i) kurzzeitig verarbeitet. Das aktualisierte COVID-Zertifikat wird
+    sodann an Ihre App zurückübermittelt.
+</p>
+<p>
+    Um eine missbräuchliche mehrfache Verwendung der Aktualisierungsfunktion verhindern zu können,
+    wird die elektronische Signatur des aktualisierten digitalen COVID-Zertifikats in Form einer
+    speziellen Einwegverschlüsselung (sog. Hash-Wert) vorübergehend auf dem Serversystem der App
+    gespeichert. Das bedeutet, dass die elektronische Signatur nach einem bestimmten mathematischen
+    Verfahren so verfremdet wird, dass sie nicht mehr erkennbar ist. Aus dem Hash-Wert kann nicht
+    auf die Daten im betreffenden digitalen COVID-Zertifikat oder andere Angaben zu Ihrer Person
+    geschlossen werden. Die eindeutige Zuordnung des Hash-Werts zu dem aktualisierten digitalen
+    COVID-Zertifikat ist aber weiterhin möglich. Nach 365 Tagen wird der Hash-Wert wieder gelöscht.
+</p>
```

Comment:
This part reads, as if the user would be able to initiate a reissuing at any time, in contrast to what is stated above.
Yes, in the end, both paragraphs can be true at the same time, independently from how the reissuance procedure is implemented into the app. But maybe a better clarification could be good here. E.g.,
"Um eine missbräuchliche mehrfache Verwendung der Aktualisierungsfunktion durch automatisierte System verhindern zu können, ..."
Then even if reissuing is implemented in the way, that "app decides whether to offer reissuing to the user or not, and to present a "temporary button" for reissuing", then it gets clear to the user, why at all it is necessary to store the DCC hash(es).